doc/mcli.md

```diff
@@ -1,5 +1,9 @@
 # Overview
 
+!!! The doc is *stale* !!!
```

Comment:
mind removing it? nobody is going to update it and there nothing worse that old docs..

Reply:
I can update it while I still remember how to use mayastor-client :)

Reply:
@tiagolobocastro : I would rather remove it too, but if you volunteer to go through the example commands, test them and update them to be accurate then I will leave it in 👍

Reply:
I've raised CAS-364 to do this